internal/dom/assets/assets.go

```diff
@@ -0,0 +1,38 @@
+// Copyright © 2019 The Homeport Team
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package assets
+
+import "github.com/homeport/pina-golada/pkg/files"
+
+//Provider is the provider instance to access pina-goladas asset framework
+var Provider ProviderInterface
+
+/*
+ProviderInterface is the interface that provides the asset file
+@pgl(package=assets&injector=Provider)
+ */
+type ProviderInterface interface {
+	/*
+	GetGoSampleApp returns the directory containing the go sample app
+	@pgl(asset=/assets/go-cf-sample/&compressor=tar)
```

Comment:
Is `compressor=tar` just the tarring or does it _compress_ the content with GZip or similar as well?

Reply:
the tar implementation does use gzip to compress it yes